css-basic2/css-basics_pokemon-page/css/styles.css

```diff
@@ -21,6 +24,9 @@
 /* Task 3: To set the line height to 1.5 is good for texts, because of accessibility concerns, but for the headings it seems too much. Let's change this by giving our headings a line height of 1.2: */
 
 /* your solution */
+h1, h2 {
```

Comment:
what do you think about just writing h? 
h {
line height: 1.5;
}

Reply:
not sure tbh, see: https://github.com/spiced-academy/chervil-web-dev/blob/main/sessions/css-basics/css-basics.md#combine-multiple-selectors-with-a-comma

Reply:
Thanks for the reference. I've learned something :)